unshaded_deploy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This looks a lot similar to deploy.sh, could we consolidate to remove duplicate logic?

Reply:
Let's maybe keep these two as separated files for now? Reason is:

1. All the other mvn release job run a bash script called `run.sh`, which calls `deploy.sh` under the hood, and `run.sh` does not take any parameter. Modify `deploy.sh` require more changes.
2. Seems no easy way to test the script without deploy to staging repo, so let's better keep the normal release pipeline untouched.